File: challenge-1/table.c
/*
* C Program to print multiplication
* table for number between 2 and 15
* input from the user
*/

#include <stdio.h>

// Prints table of given number upto 10
void print_table(int number);

// Driver code
int main()
{
	int num;
	printf("Input: ");
	scanf("%d", &num);

	// If input number is within range
	if (num >= 2 && num <= 15)
	{
		print_table(num);
	}

	return 0;
}

// Prints table of given parameter number upto 10
void print_table(int number)
{
	// Print table
	for (int i = 1; i <= 10; i++)
	{
		printf("%d x %d = %d\n", number, i, number * i);
	}
}

File: challenge-1/table.py
# Python3 program to print table of a given 
# number (between 2 and 15)

# Prints table of given number upto 10
def print_table(number):
	for i in range(1, 11):
		print(f'{number} x {i} = {number * i}')


# Global code, equivalent to main in C/C++
num = int(input("Input: "))

# Check input condition
if num >= 2 and num <= 15:
	print_table(num)
